app/src/main/java/de/danoeh/antennapod/preferences/PreferenceController.java

```diff
@@ -433,9 +436,17 @@ public void onResume() {
         setParallelDownloadsText(UserPreferences.getParallelDownloads());
         setEpisodeCacheSizeText(UserPreferences.getEpisodeCacheSize());
         setDataFolderText();
+        GpodnetPreferences.setSyncAttemptListener(() -> ui.getActivity().runOnUiThread(
```

Comment:
Well, it took me a few seconds to figure out what the listener actually does. Other devs should be familiar with the shared pref listener, so the code is a lot easier to understand.
I think we can spare the few nanoseconds a general listener costs.